index.html

```diff
@@ -22,50 +22,54 @@
       <nav class="mobile-menu">
         <h1 class="logo">My Logo</h1>
         <img class="navMenu" src="images/hamburger.png" id="trigger" alt="menu icon" />
+        <ul id="info-text">
+        <li class="text-1">Portfolio</li>
+         <li class="text-1">About</li>
+        <li class="text-1">Contact</li>
+        </ul>
```

Comment:
**OPTIONAL**  I'd suggest that you wrap each of this text in an `a` tag before wrapping them in an `li` tag doing this will help to improve your site's accessibility and make it easier for users to navigate your website 👍 👍